src/components/Adyen/Adyen.js

```diff
@@ -302,29 +301,39 @@ const Adyen = ({
       isMyAccount,
       paymentMethodsType
     );
+
     if (responseData?.id) {
       createDropInInstance(responseData, paymentMethodsType);
     }
   };
 
   const generateDropIns = () => {
-    const { paymentMethods } = publisherConfig;
+    const availablePaymentMethods = getAvailablePaymentMethods(
+      publisherPaymentMethods
+    );
+
+    if (!availablePaymentMethods.length) {
+      // use function below to handle case when session returns error -
+      // e.g publisher has only iDeal but you are not using VPN
+      setGeneralError(t('Payment methods not available'));
```

Comment:
this general error probably won't be needed - to confirm with Ania

Reply:
If this is the case when NO payment methods are available at all - then we probably should keep this error message

Reply:
It's case when there are no payment methods enabled by publisher and also when there are some methods but user is using (in array) only those not enabled